test/integration/image-component/typescript-style/test/index.test.js

```diff
@@ -10,7 +10,7 @@ describe('TypeScript Image Component with Styles', () => {
     it('should fail to build when the `style` prop is passed to <Image />', async () => {
       const { stderr, code } = await nextBuild(appDir, [], { stderr: true })
       expect(stderr).toMatch(/Failed to compile/)
-      expect(stderr).toMatch(/Property 'style' does not exist on type/)
+      expect(stderr).toMatch(/Type '"fixed"' is not assignable to type '"raw"/)
```

Comment:
This error message is super confusing. It should say something about removing `style`, right?